test/e2e/generate_systemd_test.go

```diff
@@ -15,19 +15,13 @@ var _ = Describe("Podman generate systemd", func() {
 	It("podman generate systemd on bogus container/pod", func() {
 		session := podmanTest.Podman([]string{"generate", "systemd", "foobar"})
 		session.WaitWithDefaultTimeout()
-		Expect(session).To(ExitWithError())
-	})
-
-	It("podman generate systemd bad restart policy", func() {
-		session := podmanTest.Podman([]string{"generate", "systemd", "--restart-policy", "never", "foobar"})
-		session.WaitWithDefaultTimeout()
-		Expect(session).To(ExitWithError())
```

Comment:
Another completely pointless test: this was failing with "I'm gonna check the container name first, before even checking options, and foobar doesn't exist, so this test doesn't even test what you think it's testing". Fortunately this was a dup, and the "bogus" test below was already testing this functionality.